packages/core/admin/ee/server/routes/features-routes.js

```diff
@@ -43,4 +43,14 @@ module.exports = {
       },
     },
   ],
+  'audit-logs': [
+    {
+      method: 'GET',
+      path: '/audit-logs',
+      handler: 'auditLogs.getAuditLogs',
+      config: {
+        policies: ['admin::isAuthenticatedAdmin'],
```

Comment:
Maybe we can add a TODO comment to remember to check for the right permissions here once we have them